salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -70,7 +72,29 @@ Cordon the node:
   metalk8s_cordon.node_cordoned:
     - name: {{ node_name }}
 
-{%- if not pillar.orchestrate.get('skip_draining', False) %}
+{%- if not skip_draining or cleanup_loop_devices %}
+  {%- set run_drain = not skip_draining %}
```

Comment:
Minor: It's not clear the first time you read this :smile: I would prefer to set `run_drain` at the beginning and use this variable directly in the if
```
{%- set run_drain = not pillar.orchestrate.get('skip_draining', False) %}
[...]
{%- if run_drain or cleanup_loop_devices %}
[...]
```

Reply:
If it's not clear then I'll change :+1: